### PR TITLE
MC-1229 Updates local network README

### PR DESCRIPTION
**Pull request by tsegaran**
Also adds a bash header and reduces the size of the bootstrap

### CODE REVIEW

File: tools/local-network/README.md [hunk 1]
```diff
@@ -1,14 +1,25 @@
 ## Intro
 
-This directory contains shell scripts for running a local 5-node mesh network and a `mobilecoind` instance.
+This directory contains scripts for running a local mobilecoin consensus network and a `mobilecoind` instance.
+The nodes are run in debug mode, so it is only intended to be used for development and testing purposes.
```
Comment (eranrund) [inline tools/local-network/README.md:4]: The script actually defaults to release mode.

File: tools/local-network/README.md [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
-## local-network-5.sh
+## local-network.py
+
+This script starts a local mobilecoin consensus network by launching a separate process for each consensus validator and configuring them to communicate via a default set of ports. It takes the following parameters:
+
+- (required) `--network-type` - describes the network topology, one of `dense5`, `a-b-c`, `ring5` or `ring5b`
+- (optional) `--skip-build` - does not rebuild consensus node binaries
 
-This script starts a 5 node network configured in a mesh topology. It relies on environment variables for configuration:
+It relies on environment variables for configuration:
```
Comment (eranrund) [inline tools/local-network/README.md:22]: There are some changes to the env vars.
- `CLIENT_RESPONDER_HOST` no longer exists.
- `MOB_RELEASE` defaults to 1, might be worth mentioning.
- `LEDGER_SYNC_S3` no longer exists.

File: tools/local-network/README.md (hunk 1 above)
Comment (tsegaran) [reply tools/local-network/README.md:4]: Was referring to this warning:
```
2020-04-24 19:32:17.520802775 UTC ERRO Enclave will be started in debug mode, there is no privacy, mc.local_node_id: localhost:3304, mc.app: consensus-service, mc.module: consensus_service, mc.src: consensus/service/src/bin/main.rs:74
```

Comment (eranrund) [reply tools/local-network/README.md:4]: I don't know how many people would think about the enclave mode, versus the build configuration.

Comment (tsegaran) [reply tools/local-network/README.md:4]: "SGX in emulation mode"?

Comment (sugargoat) [reply tools/local-network/README.md:4]: Yeah - maybe to disambiguate, you could say "The enclaves are built in pre-release mode, which provides debug symbols for debugging purposes."

Comment (sugargoat) [reply tools/local-network/README.md:4]: The way to make that warning go away is to build with `IAS_MODE=PROD`.

**Pull request merged**